### PR TITLE
[ODS-6467] Update summary descriptions in Admin API schema version 2

### PR DESCRIPTION
**Pull request by vimayya**

### CODE REVIEW

File: api-specifications/admin-api/admin-api-2.2.0.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 openapi: 3.0.1
 info:
   version: 2.2.0
+  title: "Ed-Fi Admin API"
```
Comment (jagudelo-gap) [inline api-specifications/admin-api/admin-api-2.2.0.yaml:4]: @stephenfuqua 
I remember you told me to remove the title, are you ok to include it again?

**Review by jasonh-edfi — APPROVED** [review]
Reviewed and approved

File: api-specifications/admin-api/admin-api-2.2.0.yaml (hunk 1 above)
Comment (vimayya) [reply api-specifications/admin-api/admin-api-2.2.0.yaml:4]: Without the title swagger editor https://editor.swagger.io/ wouldn't load the metadata

**Review by jasonh-edfi — COMMENTED** [review]
Reviewed and approved

**Pull request merged**